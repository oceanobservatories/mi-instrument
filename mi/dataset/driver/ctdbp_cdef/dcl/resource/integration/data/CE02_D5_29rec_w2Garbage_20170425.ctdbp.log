2017/04/25 12:30:13.199 [ctdbp:DLOGP3]:Instrument Started [Power On]
2017/04/25 12:30:26.450 # 10.8939,  3.64517,    7.367, 25 Apr 2017 12:30:20
2017/04/25 12:30:36.450 # 10.8936,  3.64509,    6.767, 25 Apr 2017 12:30:30
2017/04/25 12:30:46.443 # 10.8937,  3.64512     6.867, 25 Apr 2017 12:30:40
2017/04/25 12:30:56.444 # 10.8937,  3.64516,    7.358, 25 Apr 2017 12:30:50
2017/04/25 12:31:06.444 # 10.8936,  3.64508,    6.643, 25 Apr 2017 12:31:00
2017/04/25 12:31:16.447 # 10.8935,  3.64506,    6.669, 25 Apr 2017 12:31:10
2017/04/25 12:31:26.443 # 10.8932,  3.64512,    6.921, 25 Apr 2017 12:31:20
2017/04/25 12:31:46.460 # 10.8932,  3.64516,    6.836, 25 Apr 2017 12:31:40
2017/04/25 12:31:56.443 # 10.8931,  3.64525,    7.542, Apr 2017 12:31:50
2017/04/25 12:32:06.442 # 10.8930,  3.64524,    7.596, 25 Apr 2017 12:32:00
2017/04/25 12:32:16.443 # 10.8930,  3.64519,    7.386, 25 Apr 2017 12:32:10
2017/04/25 12:32:26.442 # 10.8929,  3.64515,    7.446, 25 Apr 2017 12:32:20
2017/04/25 12:32:36.443 # 10.8925,  3.64504,    6.755, 25 Apr 2017 12:32:30
2017/04/25 12:32:56.442 # 10.8928,  3.64514,    7.075, 25 Apr 2017 12:32:50
2017/04/25 12:33:06.441 # 10.8929,  3.64515,    7.541, 25 Apr 2017 12:33:00
2017/04/25 12:33:16.442 # 10.8926,  3.64514,    6.710, 25 Apr 2017 12:33:10
2017/04/25 12:33:22.929 [ctdbp:DLOGP3]:Instrument Stopped [Power Off]
2017/04/25 12:45:10.453 [ctdbp:DLOGP3]:Instrument Started [Power On]
2017/04/25 12:45:23.449 # 10.8942,  3.64532,    6.901, 25 Apr 2017 12:45:17
2017/04/25 12:45:33.442 # 10.8942,  3.64526,    6.082, 25 Apr 2017 12:45:27
2017/04/25 12:45:43.442 # 10.8943,  3.64529,    6.861, 25 Apr 2017 12:45:37
2017/04/25 12:45:53.444 # 10.8939,  3.64520,    6.883, 25 Apr 2017 12:45:47
2017/04/25 12:46:13.443 # 10.8934,  3.64520,    6.535, 25 Apr 2017 12:46:07
2017/04/25 12:46:23.454 # 10.8929,  3.64513,    6.853, 25 Apr 2017 12:46:17
2017/04/25 12:46:33.443 # 10.8930,  3.64513,    6.965, 25 Apr 2017 12:46:27
2017/04/25 12:46:43.442 # 10.8923,  3.64507,    6.708, 25 Apr 2017 12:46:37
2017/04/25 12:47:03.443 # 10.8928,  3.64512,    7.429, 25 Apr 2017 12:46:57
2017/04/25 12:47:13.442 # 10.8928,  3.64513,    7.066, 25 Apr 2017 12:47:07
2017/04/25 12:47:23.443 # 10.8927,  3.64509,    6.607, 25 Apr 2017 12:47:17
2017/04/25 12:47:33.442 # 10.8932,  3.64516,    7.300, 25 Apr 2017 12:47:27
2017/04/25 12:47:53.443 # 10.8933,  3.64514,    6.864, 25 Apr 2017 12:47:47
2017/04/25 12:48:03.442 # 10.8926,  3.64508,    6.624, 25 Apr 2017 12:47:57
2017/04/25 12:48:13.443 # 10.8927,  3.64510,    7.374, 25 Apr 2017 12:48:07
2017/04/25 12:48:19.924 [ctdbp:DLOGP3]:Instrument Stopped [Power Off]
