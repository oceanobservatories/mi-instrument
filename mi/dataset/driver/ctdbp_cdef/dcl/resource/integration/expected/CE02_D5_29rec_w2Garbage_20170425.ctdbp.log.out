
 subsite  | node  | sensor       | bin        | method      | time                  | deployment | internal_timestamp | port_timestamp        | preferred_timestamp | quality_flag | conductivity | date_time_string | dcl_controller_timestamp | pressure | temp
----------+-------+--------------+------------+-------------+-----------------------+------------+--------------------+-----------------------+---------------------+--------------+--------------+------------------+--------------------------+----------+---------
 CE02SHSM | RID27 | 03-CTDBPC000 | 3702067200 | telemetered | 3702112226.4499998093 |          5 |         3702112220 | 3702112226.4499998093 |      port_timestamp |           ok |      3.64517 |             null |                     null |    7.367 | 10.8939
 CE02SHSM | RID27 | 03-CTDBPC000 | 3702067200 | telemetered | 3702112236.4499998093 |          5 |         3702112230 | 3702112236.4499998093 |      port_timestamp |           ok |      3.64509 |             null |                     null |    6.767 | 10.8936
 CE02SHSM | RID27 | 03-CTDBPC000 | 3702067200 | telemetered | 3702112256.4440002441 |          5 |         3702112250 | 3702112256.4440002441 |      port_timestamp |           ok |      3.64516 |             null |                     null |    7.358 | 10.8937
 CE02SHSM | RID27 | 03-CTDBPC000 | 3702067200 | telemetered | 3702112266.4440002441 |          5 |         3702112260 | 3702112266.4440002441 |      port_timestamp |           ok |      3.64508 |             null |                     null |    6.643 | 10.8936
 CE02SHSM | RID27 | 03-CTDBPC000 | 3702067200 | telemetered | 3702112276.4470000267 |          5 |         3702112270 | 3702112276.4470000267 |      port_timestamp |           ok |      3.64506 |             null |                     null |    6.669 | 10.8935
 CE02SHSM | RID27 | 03-CTDBPC000 | 3702067200 | telemetered | 3702112286.4429998398 |          5 |         3702112280 | 3702112286.4429998398 |      port_timestamp |           ok |      3.64512 |             null |                     null |    6.921 | 10.8932
 CE02SHSM | RID27 | 03-CTDBPC000 | 3702067200 | telemetered | 3702112306.4600000381 |          5 |         3702112300 | 3702112306.4600000381 |      port_timestamp |           ok |      3.64516 |             null |                     null |    6.836 | 10.8932
 CE02SHSM | RID27 | 03-CTDBPC000 | 3702067200 | telemetered | 3702112326.4419999123 |          5 |         3702112320 | 3702112326.4419999123 |      port_timestamp |           ok |      3.64524 |             null |                     null |    7.596 |  10.893
 CE02SHSM | RID27 | 03-CTDBPC000 | 3702067200 | telemetered | 3702112336.4429998398 |          5 |         3702112330 | 3702112336.4429998398 |      port_timestamp |           ok |      3.64519 |             null |                     null |    7.386 |  10.893
 CE02SHSM | RID27 | 03-CTDBPC000 | 3702067200 | telemetered | 3702112346.4419999123 |          5 |         3702112340 | 3702112346.4419999123 |      port_timestamp |           ok |      3.64515 |             null |                     null |    7.446 | 10.8929
 CE02SHSM | RID27 | 03-CTDBPC000 | 3702067200 | telemetered | 3702112356.4429998398 |          5 |         3702112350 | 3702112356.4429998398 |      port_timestamp |           ok |      3.64504 |             null |                     null |    6.755 | 10.8925
 CE02SHSM | RID27 | 03-CTDBPC000 | 3702067200 | telemetered | 3702112376.4419999123 |          5 |         3702112370 | 3702112376.4419999123 |      port_timestamp |           ok |      3.64514 |             null |                     null |    7.075 | 10.8928
 CE02SHSM | RID27 | 03-CTDBPC000 | 3702067200 | telemetered | 3702112386.4409999847 |          5 |         3702112380 | 3702112386.4409999847 |      port_timestamp |           ok |      3.64515 |             null |                     null |    7.541 | 10.8929
 CE02SHSM | RID27 | 03-CTDBPC000 | 3702067200 | telemetered | 3702112396.4419999123 |          5 |         3702112390 | 3702112396.4419999123 |      port_timestamp |           ok |      3.64514 |             null |                     null |     6.71 | 10.8926
 CE02SHSM | RID27 | 03-CTDBPC000 | 3702067200 | telemetered | 3702113123.4489998817 |          5 |         3702113117 | 3702113123.4489998817 |      port_timestamp |           ok |      3.64532 |             null |                     null |    6.901 | 10.8942
 CE02SHSM | RID27 | 03-CTDBPC000 | 3702067200 | telemetered | 3702113133.4419999123 |          5 |         3702113127 | 3702113133.4419999123 |      port_timestamp |           ok |      3.64526 |             null |                     null |    6.082 | 10.8942
 CE02SHSM | RID27 | 03-CTDBPC000 | 3702067200 | telemetered | 3702113143.4419999123 |          5 |         3702113137 | 3702113143.4419999123 |      port_timestamp |           ok |      3.64529 |             null |                     null |    6.861 | 10.8943
 CE02SHSM | RID27 | 03-CTDBPC000 | 3702067200 | telemetered | 3702113153.4440002441 |          5 |         3702113147 | 3702113153.4440002441 |      port_timestamp |           ok |       3.6452 |             null |                     null |    6.883 | 10.8939
 CE02SHSM | RID27 | 03-CTDBPC000 | 3702067200 | telemetered | 3702113173.4429998398 |          5 |         3702113167 | 3702113173.4429998398 |      port_timestamp |           ok |       3.6452 |             null |                     null |    6.535 | 10.8934
 CE02SHSM | RID27 | 03-CTDBPC000 | 3702067200 | telemetered | 3702113183.4539999962 |          5 |         3702113177 | 3702113183.4539999962 |      port_timestamp |           ok |      3.64513 |             null |                     null |    6.853 | 10.8929
 CE02SHSM | RID27 | 03-CTDBPC000 | 3702067200 | telemetered | 3702113193.4429998398 |          5 |         3702113187 | 3702113193.4429998398 |      port_timestamp |           ok |      3.64513 |             null |                     null |    6.965 |  10.893
 CE02SHSM | RID27 | 03-CTDBPC000 | 3702067200 | telemetered | 3702113203.4419999123 |          5 |         3702113197 | 3702113203.4419999123 |      port_timestamp |           ok |      3.64507 |             null |                     null |    6.708 | 10.8923
 CE02SHSM | RID27 | 03-CTDBPC000 | 3702067200 | telemetered | 3702113223.4429998398 |          5 |         3702113217 | 3702113223.4429998398 |      port_timestamp |           ok |      3.64512 |             null |                     null |    7.429 | 10.8928
 CE02SHSM | RID27 | 03-CTDBPC000 | 3702067200 | telemetered | 3702113233.4419999123 |          5 |         3702113227 | 3702113233.4419999123 |      port_timestamp |           ok |      3.64513 |             null |                     null |    7.066 | 10.8928
 CE02SHSM | RID27 | 03-CTDBPC000 | 3702067200 | telemetered | 3702113243.4429998398 |          5 |         3702113237 | 3702113243.4429998398 |      port_timestamp |           ok |      3.64509 |             null |                     null |    6.607 | 10.8927
 CE02SHSM | RID27 | 03-CTDBPC000 | 3702067200 | telemetered | 3702113253.4419999123 |          5 |         3702113247 | 3702113253.4419999123 |      port_timestamp |           ok |      3.64516 |             null |                     null |      7.3 | 10.8932
 CE02SHSM | RID27 | 03-CTDBPC000 | 3702067200 | telemetered | 3702113273.4429998398 |          5 |         3702113267 | 3702113273.4429998398 |      port_timestamp |           ok |      3.64514 |             null |                     null |    6.864 | 10.8933
 CE02SHSM | RID27 | 03-CTDBPC000 | 3702067200 | telemetered | 3702113283.4419999123 |          5 |         3702113277 | 3702113283.4419999123 |      port_timestamp |           ok |      3.64508 |             null |                     null |    6.624 | 10.8926
 CE02SHSM | RID27 | 03-CTDBPC000 | 3702067200 | telemetered | 3702113293.4429998398 |          5 |         3702113287 | 3702113293.4429998398 |      port_timestamp |           ok |       3.6451 |             null |                     null |    7.374 | 10.8927

(29 rows)
